js/ui/status/keyboard.js

```diff
@@ -778,11 +821,12 @@ const InputSourceIndicator = new Lang.Class({
     Name: 'InputSourceIndicator',
     Extends: PanelMenu.Button,
 
-    _init: function() {
+    _init: function(parent, showLayout = true) {
```

Comment:
Don't think you need to pass the parent here -- it looks unused.

Reply:
It is unused here indeed, but we do pass it when instantiating for the panel: https://github.com/endlessm/gnome-shell/blob/master/js/ui/panel.js#L1190

If I want to keep my default argument I have to add it I think.

Reply:
Sounds good!